src/jrd/MetaName.h

```diff
@@ -258,6 +258,11 @@ class MetaName
 		return !word;
 	}
 
+	bool empty() const
+	{
+		return !word;
+	}
+
```

Comment:
I see no sense to have a duplication. If we're OK with STL-like `empty()`, then let's just rename `isEmpty()` across the codebase.

Reply:
It would be too big and unrelated change for this PR but I can prepare a separate one if agreed.